### PR TITLE
Sanity Tests : Execute on Firefox & Chrome & Webkit

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Sanity Tests : Execute on Firefox & Chrome & Webkit
| Type?             | improvement
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI is :green_circle: 
| UI Tests          | N/A
| Fixed issue or discussion?     | N/A
| Related PRs       | N/A
| Sponsor company   | @PrestaShopCorp

### CODE REVIEW

**Review by SD1982 — DISMISSED** [review]

File: .github/workflows/sanity.yml [hunk 1]
```diff
@@ -21,6 +21,7 @@ jobs:
     strategy:
```
Comment (boubkerbribri) [inline .github/workflows/sanity.yml:21]: Maybe we should start running on nightly and wait for stability before running all the time on sanity ? 
WDYT ?

**Review by boubkerbribri — DISMISSED** [review]
1 Comment but LGTM

File: .github/workflows/sanity.yml (hunk 1 above)
Comment (Progi1984) [reply .github/workflows/sanity.yml:21]: I think sanity, it's the first line...

It run on each PR. That allow us to be vigilant to be stable.

(And honestly, no problem on the validation on each PR).

**Comment by Progi1984** [comment]
Ping @PrestaShop/committers for validation.

File: .github/actions/ui-test/action.yml [hunk 2]
```diff
@@ -18,6 +18,10 @@ inputs:
     required: false
     description: Node Version for tests
     default: '16'
+  BROWSER:
+    required: false
+    description: Bowser for tests
+    default: 'chrome'
```
Comment (boubkerbribri) [inline .github/actions/ui-test/action.yml]:
```suggestion
    default: 'chromium'
```

Comment (Progi1984) [reply .github/actions/ui-test/action.yml]: Fixed

**Review by boubkerbribri — DISMISSED** [review]

**Review by SD1982 — DISMISSED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Review by SD1982 — APPROVED** [review]

**Review by M0rgan01 — APPROVED** [review]

**Review by SD1982 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @jolelievre for the help :heart: on you.

**Pull request merged**